techniques/flash/FLASH20.html

```diff
@@ -61,7 +61,7 @@ <h3>A thick blue border to indicate focus</h3>
             <p class="working-example">The result of this technique can be found in the <a href="../../working-examples/flash-change-styles-on-focus/">working version of A thick blue border to indicate focus</a>. </p>
             <p class="exsubhead">Code for Example 1 (ActionScript 3.0)</p>
          
-         <pre xml:space="preserve">package wcagSamples {
+         <pre>package wcagSamples {
```

Comment:
all the naked `<pre> ... </pre>` blocks in this technique could do with inner `<code> ... </code>`